clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -1089,8 +1089,8 @@ public void handleSuccessfulResponse(RequestHeader requestHeader, long now, Meta
             if (!errors.isEmpty())
                 log.warn("Error while fetching metadata with correlation id {} : {}", requestHeader.correlationId(), errors);
 
-            // Don't update the cluster if there are no valid nodes...the topic we want may still be in the process of being
-            // created which means we will get errors and no nodes until it exists
+            // Don't update the cluster if there are no valid nodes which may happened during the startup phase of
```

Comment:
may happened -> may happen
How about rephrase to `When talking to the startup phase of a broker, it is possible to receive an empty metadata set, which we should retry later.`

Reply:
make sense